typescript-selenium-webdriver/azure-pipelines.yml

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+jobs:
+    - job: 'tests_and_checks'
+
+      pool:
+          vmImage: 'ubuntu-16.04'
+
+      steps:
+          - task: NodeTool@0
+            inputs:
+                versionSpec: '10.15.3'
+            displayName: use node 10.15.3
+
+          - script: yarn install --frozen-lockfile
+            displayName: yarn install
+
+          - script: yarn test -- --ci
+            displayName: yarn test
+
+            task: PublishTestResults@2
+            inputs:
+                testResultsFiles: $(System.DefaultWorkingDirectory)/test-results/junit.xml
+                testRunTitle: typescript-selenium-webdriver
+            condition: always()
+            displayName: publish test results
+
+          - task: PublishCodeCoverageResults@1
+            inputs:
+                codeCoverageTool: Cobertura
+                summaryFileLocation: $(System.DefaultWorkingDirectory)/test-results/coverage/cobertura-coverage.xml
+                failIfCoverageEmpty: true
+            displayName: publish code coverage
+
+          - task: PublishBuildArtifacts@1
```

Comment:
This is going to fail until we actually implement a test that writes a sarif file here; maybe omit this until we get there